src/pages/Settings/SavedSearchBeta.vue

```diff
@@ -1,41 +1,54 @@
 <template>
-	<www-page>
+	<www-page
+		:gray-background="true"
+	>
+		<template #secondary>
+			<the-my-kiva-secondary-menu />
```

Comment:
Just out of curiosity, why do we need to specify the secondary menu?

Reply:
The secondary menu will provide navigation within the "logged-in" user section of the site, similar to `/portfolio/lending-stats` or `/settings`.

Reply:
The legacy page has a very outdated version of this "secondary" nav as well.

Reply:
Sounds good, thanks.

Reply:
Oh the secondary menu is the secondary navigation which is VUE-1128!